pages/technical-samples/[uuid].js

```diff
@@ -50,8 +51,10 @@ export default function TechnicalSample({
               <SampleDataItems item={sample} sources={sources}>
                 {sample.date && (
```

Comment:
Change this check to `sample.date_obtained`.

Reply:
done